docker/ds/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY bootstrap/ /var/tmp/bootstrap/
 COPY schema /var/tmp/schema
 
 # The default BASE_DN of the directory tree.
-ENV BASE_DN o=userstore
+ENV BASE_DN dc=example,dc=com
```

Comment:
do we really want example.com?   I'd prefer a generic "o=userstore" or some such thing.

Ideally organizations don't feel the need to change it